### ISSUE TITLE
`./wpt affected-tests` can list tests when `./wpt test-jobs` doesn't list stability

### ISSUE THREAD

**Issue by foolip** [web-platform-tests/wpt#13937]
An oddity discovered in https://github.com/web-platform-tests/wpt/issues/13936 which by the looks of it isn't accidental, but still probably worth addressing.

Among the PRs I looked at, these didn't trigger `stability` but did have affected tests.

#13611 and #13877 (its revert) both modified idlharness.js, but all of resources/ is excluded:
https://github.com/web-platform-tests/wpt/blob/3422a5df992675ea3e8478504e8be11ef3494b3d/tools/ci/jobs.py#L17. `./wpt affected-tests` still listed 150 affected tests, however. @lukebjerring, thoughts on this?

#13733 modified payment-request/META.yml, which is excluded, but META.yml is mentioned in a lot of tests in payment-request/. This isn't really a problem, though.

An open question is whether using `./wpt test-jobs --includes stability` is necessary at all, or if it's better to just call `./wpt affected-tests`. An upside of `./wpt test-jobs` is that it doesn't require the manifest and so is faster.

**Comment by gsnedders** [comment]
I can't find where this comes from originally (it predates the `wpt` tool IIRC), but it was because `testharness.js` caused every single testharness.js to be rerun. You can imagine that didn't work well…

**Comment by gsnedders** [comment]
Arguably, once we move stability on to something with more capacity, we *should* run everything using testharness.js for every change of it.

**Comment by jgraham** [comment]
That seems a bit academic at the moment; not only would we need more capacity we'd need some good way to figure out how many chunks to run to get the results in a reasonable amount of time (and certainly less than the timeout of the runner). Irrespective of capacity, we haven't got any good solution for that at the moment, and getting one is some way off.

**Comment by gsnedders** [comment]
@jgraham Definitely. That said, at least doing one run so regressions could be surfaced would be useful.

**Comment by gsnedders** [comment]
(Though I guess that's basically just #13263.)

**Comment by qiuzhong** [comment]
@foolip , I got this error when running `./wpt affected-tests`:
```
$ ./wpt affected-tests
usage: wpt [-h] [--venv VENV] [--debug]
           {tc-download,run,test-jobs,branch-point,create,lint,serve,check-stability,update-expectations,manifest,install,make-hosts-file,manifest-download,files-changed,tests-affected}
           ...
wpt: error: argument command: invalid choice: 'affected-tests' (choose from u'tc-download', u'run', u'test-jobs', u'branch-point', u'create', u'lint', u'serve', u'check-stability', u'update-expectations', u'manifest', u'install', u'make-hosts-file', u'manifest-download', u'files-changed', u'tests-affected')
```

I thought you mean `./wpt tests-affected` but typed the wrong order for this argument.

**Comment by foolip** [comment]
@qiuzhong, thank, I've updated the title and description.
